Tests/SWBBuildSystemTests/ClangCompilationCachingTests.swift

```diff
@@ -18,9 +18,7 @@ import SWBTaskExecution
 import SWBTestSupport
 import SWBUtil
 
-@Suite(.skipHostOS(.windows), .requireCompilationCaching,
-    .requireDependencyScannerPlusCaching, // Windows: CAS creation failed: fatal error: CAS cannot be initialized from the specified '-fcas-*' options: OnDiskCache is disabled
```

Comment:
I think this comment was supposed to be attached to the skipHostOSWindows and shouldn't be removed.

Maybe try rewriting as:

```swift
.skipHostOS(.windows, "Windows: CAS creation failed: fatal error: CAS cannot be initialized from the specified '-fcas-*' options: OnDiskCache is disabled")
```

Reply:
OK, thanks